automation_tools/satellite6/upgrade/satellite.py

```diff
@@ -66,7 +66,7 @@ def satellite6_setup(os_version):
     # Set satellite hostname in fabric environment
     env['satellite_host'] = sat_host
     logger.info('Satellite {} is ready for Upgrade!'.format(sat_host))
-    if os.environ.get('RUN_EXISTANCE_TESTS'):
+    if bool(os.environ.get('RUN_EXISTANCE_TESTS', False)):
```

Comment:
Doing this way RUN_EXISTENCE_TESTS will run even when == `false`.

Better to do this:

```python
if os.environ.get('RUN_EXISTANCE_TESTS', 'false').lower() == 'true':
   # ...
```